### PR TITLE
dotCMS/core#25316 [UI] Add to Bundle to the 3 dots Menu in Experiment Configuration

### LINKED ISSUE #25316: [UI] Add to Bundle to the 3 dots Menu in Experiment Configuration
### Parent Issue

N/A

### User Story

Add the add to Bundle option in the experiments configuration page and listing. 

### Acceptance Criteria

-  should be present in the configuration page options at the top right
-  Should be present in the listing page. 
- Add To Bundle should be present in all experiment states 
- Only present if has enterprise license

### Proposed Objective

User Experience

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Support add to bundle in the listing and configuration screen of experiments. 

#### Listing

image

#### Config 

image

#### Dialog 

image

### CODE REVIEW

File: core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts [hunk 1]
```diff
@@ -221,7 +221,14 @@ export const AllowedActionsByExperimentStatus: Record<
         DotExperimentStatus.ENDED,
         DotExperimentStatus.ARCHIVED
     ],
-    ['archive']: [DotExperimentStatus.ENDED]
+    ['archive']: [DotExperimentStatus.ENDED],
+    ['addToBundle']: [
```
Comment (jdotcms) [inline core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts:225]: I wonder if this should be not retrieve from the backend, any new state or action will need to be replicated here

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts [hunk 2]
```diff
@@ -102,7 +118,9 @@ describe('DotExperimentsConfigurationComponent', () => {
                 provide: DotMessageService,
                 useValue: messageServiceMock
             },
-
+            { provide: CoreWebService, useClass: CoreWebServiceMock },
+            { provide: DotCurrentUserService, useClass: DotCurrentUserServiceMock },
+            mockProvider(LoggerService),
```
Comment (fmontes) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts:123]: Can't we use `mockProvider` to mock `CoreWebService`?

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-portlet-routes.ts [hunk 3]
```diff
@@ -2,13 +2,17 @@ import { Routes } from '@angular/router';
...
 
 import { DotExperimentsShellComponent } from './dot-experiments-shell/dot-experiments-shell.component';
 
 export const DotExperimentsPortletRoutes: Routes = [
     {
         path: ':pageId',
         component: DotExperimentsShellComponent,
+        resolve: {
+            isEnterprise: DotEnterpriseLicenseResolver
+        },
```
Comment (fmontes) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-portlet-routes.ts:15]: WOW

**Review by fmontes — APPROVED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts (hunk 2 above)
Comment (hmoreras) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts:123]: tried, but will check again before merge it

Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/dot-experiments-configuration.component.spec.ts:123]: Let's try mock the complete `Component` if not work the mockProvider

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.spec.ts [hunk 4]
```diff
@@ -49,6 +68,7 @@ describe('ExperimentsListComponent', () => {
     const createComponent = createComponentFactory({
         component: DotExperimentsListComponent,
         componentProviders: [provideComponentStore(DotExperimentsListStore)],
+        imports: [DotAddToBundleModule, HttpClientTestingModule],
```
Comment (oidacra) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.spec.ts:71]: Try the same as the prev. Mock the module, and remove the extras providers added. And I think `HttpClientTestingModule` is not necessary.

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.spec.ts [hunk 5]
```diff
@@ -64,7 +84,10 @@ describe('ExperimentsListComponent', () => {
             {
                 provide: ActivatedRoute,
                 useClass: ActivatedRouteListStoreMock
-            }
+            },
+            { provide: CoreWebService, useClass: CoreWebServiceMock },
+            { provide: DotCurrentUserService, useClass: DotCurrentUserServiceMock },
```
Comment (oidacra) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.spec.ts:89]: same

**Review by oidacra — APPROVED** [review]

**Pull request merged**